iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -538,7 +539,7 @@ private boolean canProvideColStats(Table table, long snapshotId) {
     try {
       FileSystem fs = statsPath.getFileSystem(conf);
       return  fs.exists(statsPath);
-    } catch (IOException e) {
+    } catch (Exception e) {
```

Comment:
I think we need catch all exception to avoid failing here, as colStas should not stop we querying the table.